.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,78 @@
+name: "🐞 Bug report"
```

Comment:
Should the extension be `YAML` or `YML`?

Reply:
Yeah, I just remembered our YAMLs to be `.yml`
Never mind, if it works it works